playbooks/fix-mysql-gcache.yml

```diff
@@ -0,0 +1,43 @@
+---
+- hosts: controller
+  tasks:
+    - name: determine if it's a secondary node
+      shell: ip a | grep ucarp > /dev/null 2>&1
+      register: ucarp_result
+      ignore_errors: true
+
+    - name: stop mysql service on secondary node
+      service: name=mysql state=stopped 
+      when: ucarp_result.rc == 1
+
+    - name: clean existing gcache.page files on secondary node
+      shell: rm -rf /var/lib/mysql/gcache.page.*
+      when: ucarp_result.rc == 1
+
+    - name: start mysql service on secondary node
+      service: name=mysql state=started 
+      when: ucarp_result.rc == 1
+
+    - name: wait 10 minutes
```

Comment:
it's based on RTC 187425 related comment 2, seems operation team use this manual steps to clean the gcache pages. I guess the reason for the waiting is let 2nd node has enough time to sync the data from primary node.